adoc/chapters/programming_interface.adoc

```diff
@@ -6501,7 +6501,8 @@ accessor()
 --
   * [code]#(empty() == true)#
   * All size queries return [code]#0#.
-  * The only iterator that can be obtained is [code]#nullptr#.
+  * The return values of [code]#get_pointer()# and [code]#get_multi_ptr()# are
```

Comment:
Why unspecified instead of `nullptr`?

Reply:
Remember, there are other ways to create an empty accessor.  For example, you could create a ranged accessor with an empty range.  I think we should make the same guarantee about `get_pointer()` for all empty accessors, regardless of how they were created.  Do want to guarantee that `get_pointer()` returns `nullptr` even when a ranged accessor is empty?  It wouldn't be too hard to implement this, but it seems like extra code in the implementation that doesn't really provide that much benefit.  Why is it important to guarantee that `get_pointer()` returns `nullptr` for an empty accessor?

Somewhat related is this description in [4.7.6.6. "Accessing elements of an accessor"](https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#_accessing_elements_of_an_accessor):

> Any attempt to access an element via an index that is outside of this range produces undefined behavior.

Since there is no valid index for an empty accessor, this means that any attempt to access an element of an empty accessor produces undefined behavior.  To me, that wording seemed consistent with saying that the value of `get_pointer()` is undefined for an empty accessor.

Reply:
Sounds good to me, thanks for the clarification.